src/uu/printf/Cargo.toml

```diff
@@ -1,27 +1,27 @@
 [package]
-name = "uu_printf"
-version = "0.0.12"
 authors = [
-    "Nathan Ross",
-    "uutils developers",
+  "Nathan Ross",
+  "uutils developers",
 ]
-license = "MIT"
 description = "printf ~ (uutils) FORMAT and display ARGUMENTS"
+license = "MIT"
```

Comment:
I am a bit confused why you are moving stuff ? :)

Reply:
ah, that's not me, well not directly, it's the vscode format plugin for Cargo/Rust

Reply:
if you don't mind, yeah :)

Reply:
I hope the whitespace fix-ups are acceptable